src/index.html

```diff
@@ -7,8 +7,89 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@400;500;700&amp;family=Roboto:wght@400;500;700;900&amp;display=swap" rel="stylesheet">
   </head>
+  <header class="header-content">
+    <a
+      href="#home"
+      class="logo"
+    >
+      <img
+        src="./images/logo.png"
+        alt="logo images"
+      >
+    </a>
+    <nav class="menu-moyo">
+      <ul class="menu-moyo-list">
+        <li class="nav-moyo-item">
+          <a
+            href="#apple"
+            class="nav-moyo-link
+            is-active"
+          >
+          Apple
+          </a>
+        </li>
+        <li class="nav-moyo-item">
```

Comment:
add an empty line between _multiline_ siblings
```suggestion
    </a>
    
    <nav class="menu-moyo">
      <ul class="menu-moyo-list">
        <li class="nav-moyo-item">
          <a
            href="#apple"
            class="nav-moyo-link
            is-active"
          >
          Apple
          </a>
        </li>
        
        <li class="nav-moyo-item">
```